API/RATMain.m

```diff
@@ -34,6 +34,8 @@
             fprintf('\nRunning DREAM\n\n');
         end
         [problemStruct,result,bayesResults] = runDREAM(problemStruct,problemCells,problemLimits,controls,priors);
+    otherwise
```

Comment:
This should work for enums

`error('The procedure "%s" is not supported. The procedure must be one of "%s"', controls.procedure, strjoin(fieldnames(coderEnums.procedures), '", "'));`

Reply:
I was trying that and having the build fail. I'll give this a go and see if it works.

Reply:
That seems to build, so we're all good.